signature/policy_config.go

```diff
@@ -51,28 +52,44 @@ func (err InvalidPolicyFormatError) Error() string {
 // NOTE: When this function returns an error, report it to the user and abort.
 // DO NOT hard-code fallback policies in your application.
 func DefaultPolicy(sys *types.SystemContext) (*Policy, error) {
-	return NewPolicyFromFile(defaultPolicyPath(sys))
+	policyPath, err := defaultPolicyPath(sys)
+	if err != nil {
+		return nil, err
+	}
+	return NewPolicyFromFile(policyPath)
 }
 
 // defaultPolicyPath returns a path to the default policy of the system.
```

Comment:
If this now fails if the file is missing, that needs to be documented here.

Reply:
I didn't get it documented here means ?? 

I’m still getting familiar with this repository as I’m a beginner and new to contributing here. Apologies if my questions are basic.

Reply:
Right now that comment says “returns a path to the default”. So, something like “returns a path to the relevant policy of the system, or an error if the policy is missing”.

Reply:
Thanks @mtrmac ,, Updated document as per above comment.